src/components/PatronDetail/PatronDetail.test.js

```diff
@@ -15,21 +35,48 @@ describe('PatronDetail', () => {
       },
       barcode: 'barcode',
       expirationDate: 'expirationDate',
+      patronGroup: 'patronGroup',
+      type: 'userType',
     },
     isUserProfilePicConfigEnabledForTenant: true,
   };
 
-  it('should render PatronDetail', () => {
-    render(<PatronDetail {...props} />);
-    expect(screen.getByText('lastName, firstName')).toBeDefined();
+  [
+    { id: 'ui-reading-room.userDetail.firstName', value: 'First name' },
+    { id: 'ui-reading-room.userDetail.lastName', value: 'Last name' },
+    { id: 'ui-reading-room.userDetail.patronGroup', value: 'Patron Group' },
+    { id: 'ui-reading-room.userDetail.userType', value: 'User Type' },
+    { id: 'ui-reading-room.userDetail.barcode', value: 'Barcode' },
+    { id: 'ui-reading-room.userDetail.expiration', value: 'Expiration' },
+  ].map((item) => (
+    it(`should render ${item.value}`, () => {
```

Comment:
Jest supports `it.each` where you can pass an array of test data and it will run many tests.
Also the advantage of this is - if one test fails in Array.map then the rest of the tests in that group won't run. But with `it.each` if one test fails it won't stop other tests from executing